src/bodypart.cpp

```diff
@@ -106,6 +106,30 @@ bool is_legacy_bodypart_id( const std::string &id )
     return false;
 }
 
+const std::string &bodypart_str_id_to_readable( const bodypart_str_id &id )
+{
+    static const std::map<std::string, std::string> bodyparts = {
```

Comment:
Why are you having this be string to string instead of id to string? You've got the ids in here, there's no real need to convert them to strings before you map them.